mod/profiles/set/self/wikirate_company/factories.rb

```diff
@@ -39,10 +42,10 @@ def search_by_country
   end
 
   def country_code
-    Env.params[:country_code]
+    Env.params[:country_code] if Env.params[:country_code].present?
   end
 
   def keyword
-    Env.params[:keyword]
+    Env.params[:keyword] if Env.params[:keyword].present?
```

Comment:
would be nice to have a shortcut for this.  But you can just use `param` in format objects.  (see def params in format.rb)

Reply:
I always forget if it's defined in format or card objects so I always use `Env` to be safe.